src/screens/Product/Product.js

```diff
@@ -16,16 +54,34 @@ class Product extends Component {
               <Text style={styles.bodyText}>{'\t\t\t'}{textObj.value}</Text>
             </View>}
             <View style={styles.buttonsBlock}>
+              <CustomButton
```

Comment:
It would be nice if this button also could show indicator on press. But is is up to you)